packages/common/src/workflow-options.ts

```diff
@@ -39,13 +39,57 @@ export enum WorkflowIdReusePolicy {
 
   /**
    * Terminate the current workflow if one is already running.
+   *
+   * This option belongs in `WorkflowIdConflictPolicy` but it is here for backwards compatibility.
+   * With no Running Workflow, the behavior is the same as `WORKFLOW_ID_REUSE_POLICY_ALLOW_DUPLICATE`,
+   * but with a Running Workflow, it behaves like `WORKFLOW_ID_CONFLICT_POLICY_TERMINATE_EXISTING`.
+   *
+   * *Note: When setting this option, `WorkflowIdConflictPolicy` should have value `WORKFLOW_ID_CONFLICT_POLICY_UNSPECIFIED`*
    */
   WORKFLOW_ID_REUSE_POLICY_TERMINATE_IF_RUNNING = 4,
 }
 
 checkExtends<temporal.api.enums.v1.WorkflowIdReusePolicy, WorkflowIdReusePolicy>();
 checkExtends<WorkflowIdReusePolicy, temporal.api.enums.v1.WorkflowIdReusePolicy>();
 
+// Avoid importing the proto implementation to reduce workflow bundle size
+// Copied from temporal.api.enums.v1.WorkflowIdConflictPolicy
+/**
+ * Defines what should happen when trying to start a Workflow with the same Workflow Id of a Running workflow.
+ *
+ * See `WorkflowIdReusePolicy` for handling Workflow Id duplication of a *Closed* workflow.
+ *
+ * *Note: It is never valid to have two actively running instances with the same Workflow Id.*
+ */
+export enum WorkflowIdConflictPolicy {
```

Comment:
Just minor suggestions for consistency with the other one:

```suggestion
/**
 * Defines what happens when trying to start a Workflow with the same ID as a *Running* Workflow.
 *
 * See {@link WorkflowOptions.workflowIdReusePolicy} for what happens when trying to start a Workflow with the same ID as a *Closed* Workflow.
 *
 * *Note: It is not possible to have two actively running Workflows with the same ID.*
 */
export enum WorkflowIdConflictPolicy {
```